.github/actions/tool_tests/action.yaml

```diff
@@ -31,6 +31,10 @@ runs:
       shell: bash
       working-directory: ${{ inputs.path }}
 
+    - name: Install packages
+      run: sudo apt-get -y install libperl-critic-perl perltidy
+      shell: bash
```

Comment:
Go ahead and remove this. Tool tests are separate, and since this is only integrating linters/formatters, it's not needed here

Reply:
Done.